ITEvents/Screens/EventCollectionViewHandler.swift

```diff
@@ -78,8 +78,8 @@ class EventCollectionViewHandler: NSObject, IEventCollectionViewHandler {
     }
     
     func collectionView(_ collectionView: UICollectionView, willDisplay cell: UICollectionViewCell, forItemAt indexPath: IndexPath) {
-        let lastEventIndex = events.count - 1
-        if indexPath.row == lastEventIndex && !isLoadingIndicatorShown {
+        let eventIndex = events.count - 3
+        if indexPath.row == eventIndex && !isLoadingIndicatorShown {
             lastCellWillDisplayAction?()
```

Comment:
We are no longer using **last** cell so it'd better to do a couple of renamings:
- ```eventIndex``` -> ```thresholdEventIndex``` 
- ```lastCellWillDisplayAction``` -> ```thresholdCellWillDisplayAction```


```swift
func collectionView(_ collectionView: UICollectionView, willDisplay cell: UICollectionViewCell, forItemAt indexPath: IndexPath) {
   if (isLoadingIndicatorShown) {
       return
   }
   
   let thresholdEventIndex = events.count - 3
   if indexPath.row == thresholdEventIndex {
       thresholdCellWillDisplayAction?()
   }
}
```